CVE-2013-7348.yml

```diff
@@ -0,0 +1,375 @@
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+    The automated, contest, and developer flags can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then please
+    explain where you looked.
+  automated: nil
+  developer: nil
+discussion:
+  note: https://mirrors.edge.kernel.org/pub/linux/kernel/v3.x/ChangeLog-3.0.10
+  question: |
+   How was this vulnerability discovered?
+
+    Go to the bug report and read the conversation to find out how this was
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+
+    Example include:
+      * Is this out of our scope?
+      * Is this a security?
+      * How should we fix this?
+
+    Just because you see multiple comments doesn't mean it's a discussion.
+    For example:
+      * "Fix line 10". "Ok" is not what we call a discussion
+      * "Ping" (reminding people)
+
+    Check the bugs reports, pull requests, and mailing lists archives.
+
+    These answers should be boolean.
+      discussed_as_security: true or false
+      any_discussion: true or false
+
+    Put any links to disagreements you found in the notes section, or any other
+    comment you want to make.
+  any_discussion: True
+  discussed_as_security: True
+stacktrace:
+  note:
+  question: |
+    Are there any stacktraces in the bug reports?
+
+    Secondly, if there is a stacktrace, is the fix in the same file that the
+    stacktrace points to?
+
+    If there are no stacktraces, then both of these are false - but be sure to
+    mention where you checked in the note.
+
+    Answer must be true or false.
+    Write a note about how you came to the conclusions you did, regardless of
+    what your answer was.
+  any_stacktraces:
+  stacktrace_with_fix:
+description:
+unit_tested:
+  fix:
+  code:
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+
+    For code: and fix: - your answer should be boolean.
+
+    For the code_answer below, look not only at the fix but the surrounding
+    code near the fix in related directories and determine if and was there were
+    unit tests involved for this subsystem.
+
+    For the fix_answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+  fix_answer:
+  code_answer:
+reported_date:
+specification:
```

Comment:
May have just forgot this one. The  reasoning doesn't have to be too long but I do think you should have it